resources/idlharness.js

```diff
@@ -840,9 +840,9 @@ IdlInterface.prototype.test_self = function()
         var desc = Object.getOwnPropertyDescriptor(self, this.name);
         assert_false("get" in desc, "self's property " + format_value(this.name) + " has getter");
         assert_false("set" in desc, "self's property " + format_value(this.name) + " has setter");
-        assert_true(desc.writable, "self's property " + format_value(this.name) + " is not writable");
+        assert_true(desc.writable, "self's property " + format_value(this.name) + " is writable");
         assert_false(desc.enumerable, "self's property " + format_value(this.name) + " is enumerable");
-        assert_true(desc.configurable, "self's property " + format_value(this.name) + " is not configurable");
+        assert_true(desc.configurable, "self's property " + format_value(this.name) + " is configurable");
```

Comment:
Yeah, the description needs to match the expectation. See http://web-platform-tests.org/writing-tests/testharness-api.html. We could document that better though.